addons/info/src/components/Story.js

```diff
@@ -1,10 +1,10 @@
-import PropTypes from 'prop-types';
-import React from 'react';
-import MTRC from 'markdown-to-react-components';
-import PropTable from './PropTable';
-import Node from './Node';
-import { baseFonts } from './theme';
-import { Pre } from './markdown';
+import PropTypes from 'prop-types'
```

Comment:
sorry, you linted everything right, and I made you revert it... Luckely linting will soon be fixed! https://github.com/storybooks/storybook/pull/1000